chapters/clears.txt

```diff
@@ -155,20 +155,20 @@ ifndef::VK_VERSION_1_2,VK_EXT_separate_stencil_usage[]
 endif::VK_VERSION_1_2,VK_EXT_separate_stencil_usage[]
 ifdef::VK_VERSION_1_2,VK_EXT_separate_stencil_usage[]
   * [[VUID-vkCmdClearDepthStencilImage-pRanges-02658]]
-    If any element of pname:pRanges.aspect includes
+    If pname:aspect member of any element of pname:pRanges includes
```

Comment:
"If the aspect member..." here and below.

Reply:
done